module-api/src/main/java/ru/feathercore/moduleapi/ModuleLoader.java

```diff
@@ -62,4 +62,15 @@
      * @apiNote modifications to returned collection should not have side-effect on this module loader
      */
     Collection<? extends M> getModules();
+
+    /**
+     * Checks whether module of given class was loaded by this module loader.
```

Comment:
Checks whether *the* module of given *type* is *currently* loaded by this module loader.`